spectacles/cli.py

```diff
@@ -209,6 +209,10 @@ def preprocess_dashes(args: List[str]) -> List[str]:
     return processed
 
 
+def process_import_refs(input: List[str]) -> dict:
+    return {n.split(":")[0]: n.split(":")[1] for n in input}
```

Comment:
```suggestion
    return dict(arg.split(":") for arg in input)
```

Reply:
We should have some error handling here in case the format is wrong. Try/catch on the right exception.